app/assets/javascripts/oxalis/model/sagas/isosurface_saga.js

```diff
@@ -128,18 +137,22 @@ function* ensureSuitableIsosurface(): Saga<void> {
     return;
   }
   const position = yield* select(state => getFlooredPosition(state.flycam));
-  const existentMagnifications = layer.resolutions.map(resolution => Math.max(...resolution));
-  const preferredZoomStep = 1;
-  const zoomStep = Utils.clamp(
-    Math.min(...existentMagnifications),
-    preferredZoomStep,
-    Math.max(...existentMagnifications),
-  );
+  const { resolutions } = layer;
+  const preferredZoomStep = window.__isosurfaceZoomStep != null ? window.__isosurfaceZoomStep : 1;
+  const zoomStep = Math.min(preferredZoomStep, resolutions.length);
```

Comment:
The zoomstep is starting at 0 and is used as an index into the resolutions array, right? Shouldn't this be `Math.min(preferredZoomStep, resolutions.length - 1)` then?